src/components/tables/DataTable.js

```diff
@@ -141,10 +160,21 @@ export default {
     },
     search () {
       this.page = 1
+    },
+    page () {
+      this.update()
```

Comment:
update is getting called multiple times, not sure if necessary. when ```rowsPerPage``` changes, it can call update, and will change ```this.page```, which also triggers reactivity and calls ```this.update()```